mbed-cli/create-module.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -eu
+
+
+
+MBED_APP='mbed-os-example-tls'
+
+# make sure we're running in our own directory
+if [ -f create-module.sh ]; then :; else
+    cd $( dirname $0 )
+    if [ -f create-module.sh ]; then :; else
+        echo "Please run the script from is directory." >&2
+        exit 1
+    fi
+fi
+
+REMOTE_URL=$(dirname `git remote -v | grep fetch | grep ARMmbed | awk '{print $2}'`)
+MBED_APP_URL=$REMOTE_URL/$MBED_APP 
+
+mbed import $MBED_APP_URL
+
+echo "'$MBED_APP_URL' imported to '$PWD'."
```

Comment:
Is it necessary to put this code in an entirely separate script instead of just packaging it in a shell "function" `inside test/scripts/mbed-build.sh`?